provider-management.md

```diff
@@ -10,16 +10,17 @@ This module lets you manage Providers by providing actions such as **Creating, D
 On opening the Provider Management, you are greeted with a screen with active providers and a **Floating Action Button**. **Click** on the **Floating Action Button** and it will take you to new screen asking for Provider details. 
 
 ![Confirmation while creating new Provider](assets/provider_create_confirmation.png)
-The details are then matched with other existing Providers and a dialog box is shown for confirmation.
+The details are then matched with other existing Providers and a dialog box is shown for confirmation.If the device is online the provider gets synced to the servers otherwise your created provider gets saved offline and gets synced later.
 
 #### Update Providers
 
-To update a Provider, **swipe left** on the Provider to reveal additional actions (Update and Delete). **Click** on the **edit** button. It will take you to a screen where the details can be edited.
+To update a Provider, **swipe left** on the Provider to reveal additional actions (Update and Delete). **Click** on the **edit** button. It will take you to a screen where the details can be edited.If the device is online the provider gets synced to the servers otherwise your edited provider gets saved offline and gets synced later.
```

Comment:
```suggestion
To update a Provider, **swipe left** on the Provider to reveal additional actions (Update and Delete). **Click** on the **edit** button. It will take you to a screen where the details can be edited. If the device is online the provider gets synced to the servers otherwise your edited provider gets saved offline and gets synced later.
```